### DISCUSSION TITLE
Missing the `Exclude Folder From Search` context menu option

### DISCUSSION THREAD

**Discussion by ArturoDent** [microsoft/vscode-discussions#239] [Extension Development QnA] [answered]
Is it just me but I can't see a `Exclude Folder From Search` context menu option for a folder in the Explorer in Stable or Insiders' Build?

**Comment by gjsjohnmurray** [comment, marked as answer]
Per the 1.73 release notes (emphasis added):

> When right-clicking a folder in the **Search view's tree view** of results...

You need to be in Search view, not Explorer view. And you must be viewing the search results as a tree rather than as a list. Use the button in the Search view's title to select this mode.
